### ISSUE TITLE
Bump CAPI controllers to support v30 releases

### ISSUE THREAD

**Issue by yulianedyalkova** [giantswarm/roadmap#3779]
Also ensure all of our MCs are actually within the range speicified [here](https://github.com/kubernetes-sigs/cluster-api/releases).

**Comment by yulianedyalkova** [comment]
CAPA is currently blocked on upgrading the MCs to v26+ ([issue](https://github.com/giantswarm/giantswarm/issues/32462)).

CAPZ on bumping the actual controllers by Phoenix.

CAPV and CAPVCD should be OK to continue with as all MCs are on newer versions.

**Comment by yulianedyalkova** [comment]
Duplicate of https://github.com/giantswarm/roadmap/issues/3661.
